compilator/compilator_test.go

```diff
@@ -49,7 +50,11 @@ func TestMain(m *testing.M) {
 func TestCompilationEmpty(t *testing.T) {
 	assert := assert.New(t)
 
-	c, err := NewCompilator(nil, "", "", "", "", false, ui)
+	metrics := "test-metrics.csv"
```

Comment:
Just dropping the file in the working directory is kind of bad :| Make a tempfile?